ddtrace/contrib/grpc/client_interceptor.py

```diff
@@ -0,0 +1,71 @@
+import grpc
+import collections
+
+from ddtrace import Pin
+
+class _ClientCallDetails(
+        collections.namedtuple(
+            '_ClientCallDetails',
+            ('method', 'timeout', 'metadata', 'credentials')),
+        grpc.ClientCallDetails):
+    pass
+
+class _GrpcClientInterceptor(
```

Comment:
This doesn't need to be private, `GrpcClientInterceptor` is totally fine

Reply:
Would you mind adding a comment above this with a link to https://github.com/grpc/grpc/blob/d0cb61eada9d270b9043ec866b55c88617d362be/examples/python/interceptors/headers/generic_client_interceptor.py#L19

Will make it easy to jump to if/when we have to come back here.